worklist/wikipedia_utils.py

```diff
@@ -9,11 +10,14 @@ def convert_article_titles_into_ids(titles):
 
     page_title_to_id_mapping = defaultdict(int)
 
-    titles_list = ' | '.join(str(page) for page in titles)
+    for i in range(math.ceil(len(titles) / 50)):
+        # Since at max 50 titles can be passed in one request
+        titles_list = ' | '.join(str(page) for page in
+                                 titles[i * 50:(i + 1) * 50])
```

Comment:
Alternatively:

```python
chunk = articles
while chunk:
  titles_list = '|'.join(chunk[:50])
  chunk = chunk[50:]
  ...
```

would also work I think, and it's slightly easier to read. Whichever you prefer.